raiden/tests/integration/network/transport/test_udp.py

```diff
@@ -69,7 +69,7 @@ def test_udp_unreachable_node(raiden_network, skip_if_not_udp):
 @pytest.mark.parametrize('number_of_nodes', [1])
 @pytest.mark.parametrize('channels_per_node', [0])
 @pytest.mark.parametrize('number_of_tokens', [1])
-def test_suite_survives_unhandled_exception(raiden_network):
+def test_suite_survives_unhandled_exception(raiden_network, skip_if_parity):
```

Comment:
`skip_if_parity` should be a marker imho

Reply:
Same problem as with `skip_if_not_udp` and `skip_if_not_matrix` - a marker has no access to the fixtures and cannot see the blockchain type.